### PR TITLE
Feature/creation styling

### PR DESCRIPTION
**Pull request by gfodor**
Styling for homepage, refactor a bunch of SASS (WIP)

### CODE REVIEW

File: src/react-components/avatar-selector.js [hunk 1]
```diff
@@ -1,124 +1,113 @@
-import React, { Component } from 'react';
```
Comment (gfodor) [inline src/react-components/avatar-selector.js]: these changes are 99% just prettier + linter autofixes

Comment (brianpeiris) [reply src/react-components/avatar-selector.js]: 😬 thanks

File: src/react-components/hub-create-panel.js [hunk 2]
```diff
@@ -48,37 +60,104 @@ class HubCreatePanel extends Component {
...
+
+                if (this.state.expanded) {
+                  this.setState({ name: generateHubName() });
+                } else {
+                  this.setState({ expanded: true });
+                }
+              }}
+            >
+              {this.state.expanded ? (
+                <img className="create-panel__form__rotate_button" src="../assets/images/dice_icon.svg" />
```
[the remark this answers is not in the document]
Comment (gfodor) [reply src/react-components/hub-create-panel.js]: /shrug will fix

File: src/assets/stylesheets/hub-create.scss [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+	text-align: center;
+      }
+
+      @media (max-width: 520px) {
+	padding-left: 15px;
+	font-size: 1.2em;
+      }
+      
+      &--expanded {
+	padding-top: 365px;
```
[the remark this answers is not in the document]
Comment (gfodor) [reply src/assets/stylesheets/hub-create.scss]: ah nice one, this was burning me and this makes much more sense. can remove the `z-index`'es probably now

File: src/assets/stylesheets/hub-create.scss [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+      height: 95px;
+      left: 4px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      cursor: pointer;
+      bottom: 0;
+
+      @media (max-width: 520px) {
+	display: none;
```
Comment (brianpeiris) [inline src/assets/stylesheets/hub-create.scss]: Hmm, this hides the button on my S8. Is that intentional?

Comment (gfodor) [reply src/assets/stylesheets/hub-create.scss]: yes unfortunately it ends up truncating the text field too much. this means when you create a room on your phone you can't pick the scene. (yes this is bad.) what i probably should do is expand the thing automatically via a viewport check at runtime. what do you think?

Comment (gfodor) [reply src/assets/stylesheets/hub-create.scss]: actually I might be able to refactor everything to be CSS-based for expansion so maybe can use media queries.

Comment (brianpeiris) [reply src/assets/stylesheets/hub-create.scss]: The randomize button is still hidden after you expand it. Maybe we could move the text input on to its own row and have the randomize and submit button below it.
Not needed for this PR though.

Comment (gfodor) [reply src/assets/stylesheets/hub-create.scss]: I put in the hack for now so it expands on small screens. Def should fix later though.

**Pull request merged**